### DISCUSSION TITLE
Why isn't this wonderful keyboard available in official repos of any Linux distro.

### DISCUSSION THREAD

**Discussion by khaalidrafi** [OpenBangla/OpenBangla-Keyboard#198] [General]
Why isn't this keyboard available in official repo of Debian, Ubuntu, Arch, Gentoo etc.? It should be.

**Comment by bdeshi** [comment]
> Why isn't this keyboard available in official repo of Debian, Ubuntu, Arch, Gentoo etc.? It should be.

we are working on this.

**Comment by khaalidrafi** [comment]
Waiting to see it in Arch and Gentoo repos. Hope, I can use it from there soon. :)

**Reply by ahmubashshir** [reply]
It's in AUR and [chaotic-aur](https://aur.chaotic.cx/).
Let's vote it in aur so that a TU adopts it.

[aur](https://aur.archlinux.org/packages/openbangla-keyboard)
[aur-git](https://aur.archlinux.org/packages/openbangla-keyboard-git)

**Reply by khaalidrafi** [reply]
I already voted it. Can you please make an announcement to users so that they vote it? Draw their attention.

**Reply by ahmubashshir** [reply]
@mominul can you ask users to vote the aur package in readme if they use it? I think more than 9 people on arch uses obk :3

**Reply by khaalidrafi** [reply]
Also tell them to install pkgstats. It will report the number of users and 1% of users is also accepted as "popularity"

**Reply by mominul** [reply]
How about posting it on Facebook Linux forums also?

**Reply by khaalidrafi** [reply]
Bangla Linux groups and tech groups will help better.

**Comment by mominul** [comment]
I have posted these two announcements asking for voting our [AUR repository](https://aur.archlinux.org/packages/openbangla-keyboard):
https://www.facebook.com/986600591508810/posts/1963165163852343/
https://m.facebook.com/groups/linux.loverz/permalink/3674173556003579/

**Reply by khaalidrafi** [reply]
Now that it has got more than 10 votes, let's ask a trusted user to adopt it?

**Reply by ahmubashshir** [reply]
ok :P
